feature_engine/transformation/boxcox.py

```diff
@@ -139,6 +139,10 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
         # check input dataframe and if class was fitted
         X = super().transform(X)
 
+        # check contains zero or negative values
```

Comment:
I think I modified this in the main branch already, did I not? it was making the tests fail. 

In any case, I don't think we should change the boxcox in this PR. Could you please remove?

Reply:
removed it with rebase